src/AcceptanceTestsHolder/App_Packages/NSB.AcceptanceTests/Outbox/When_clearing_saga_timeouts.cs

```diff
@@ -61,18 +60,17 @@ public Task Handle(SignalDone message, IMessageHandlerContext context)
                 }
             }
 
-            public class PlaceOrderSaga : SqlSaga<PlaceOrderSaga.PlaceOrderSagaData>, IAmStartedByMessages<PlaceOrder>
+            public class PlaceOrderSaga : Saga<PlaceOrderSaga.PlaceOrderSagaData>, IAmStartedByMessages<PlaceOrder>
```

Comment:
as above, don't we need to ensure it works the same way for both?

so keep the old one as is plus add a new test for core saga?

Reply:
No, we don't need to run any given integration style test twice. Sagas in SqlP boil down to their basic details: name, correlation id, transitional correlation id, table suffix. Basically, do saga metadata. As long as we have tests that show both styles of saga boil down to the same representation of how SqlP understands sagas (and we do) we don't need to test all of NServiceBus's possible behaviors both ways.